apps/aecore/include/blocks.hrl

```diff
@@ -1,6 +1,6 @@
 -include("pow.hrl").
 
--define(PROTOCOL_VERSION, 22).
```

Comment:
I've already bumped the version once for the next release, no point doing it multiple times?

Reply:
I'm not sure, isn't this needed for integration envs? Isn't the most recent `master` automatically deployed somewhere and we need a hard fork? I don't know. @lucafavatella ?

Reply:
I see protocol version already bumped in master compared to tag v0.20.0 (protocol version bump is needed between releases e.g. between v0.20.0 and to-be-tagged v0.21.0).

In order to make our environments use a fresh DB folder in deployment of master, you need to bump deployment/DB_VERSION